src/registry.js

```diff
@@ -1,7 +1,12 @@
 const fs = require('fs');
 const validateJson = require('./validate-json-schema');
 
-const REGISTRY_JSON_FILE = './registry.json';
+const REGISTRY_JSON_FILE = 'registry.json';
+
+const TEMPLATE_STATUS_IN_VERIFICATION = 'InVerification';
+const TEMPLATE_STATUS_APPROVED = 'Approved';
+const TEMPLATE_STATUS_REJECTED = 'Rejected';
```

Comment:
It also makes sense to use enum type instead of strings for constants

Reply:
> It also makes sense to use enum type instead of strings for constants

@VladimirZaets , I didn't get your point, sorry. Javascript doesn't have the `enum` type. What did you mean?